src/components/textarea.js

```diff
@@ -1,5 +1,18 @@
 var fs = require('fs');
 module.exports = function(app) {
+  app.directive('ckeditor', function() {
```

Comment:
this causes a name collision with the other ```ckeditor``` directive that we use. I would prefer to call this ```formio-ckeditor```

Reply:
This would make then name of this become ```formioCkeditor```, which would then be implemented as ```formio-ckeditor``` in the files.